### PR TITLE
Update Github Actions Workflows

### PR DESCRIPTION
**Pull request by wanhakim**
finalized workflows
- manual-docker-build.yml
- manual-docker-scan.yml
- manual-docker-publish.yml
- pr-code-scan.yml
- pr-e2e-test.yml
- weekly-trellix-scan.yml
- nightly-e2e-test.yml

### CODE REVIEW

File: .github/workflows/manual-docker-build.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Manual - Docker Build
```
Comment (chensuyue) [inline .github/workflows/manual-docker-build.yml:4]: `Manual - Docker Build` --> `Manual - Docker Build and Test`

File: .github/workflows/manual-docker-publish.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        description: "Tag list apply to publish images, like [latest,1.0]"
+        required: false
+        type: string
+
+permissions: read-all
+jobs:
+  publish:
+    strategy:
+      matrix:
+        image: ["studio-frontend", "studio-backend", "app-frontend", "app-backend"]
```
Comment (chensuyue) [inline .github/workflows/manual-docker-publish.yml:29]: Support to set the image list as an input param, sometimes you may want to only refresh one of the images.

**Review by chensuyue — APPROVED** [review]

File: .github/workflows/manual-docker-build.yml (hunk 1 above)
Comment (wanhakim) [reply .github/workflows/manual-docker-build.yml]: updated the workflow name

File: .github/workflows/manual-docker-publish.yml (hunk 2 above)
Comment (wanhakim) [reply .github/workflows/manual-docker-publish.yml]: updated support for having the image as boolean input parameter

**Review by OngSoonEe — APPROVED** [review]

**Pull request merged**